include/AtomicDataReader.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef __READER_H__
+#define __READER_H__
+
+#include <string>
+#include <vector>
+
+class AtomicDataReader {
+public:
+  AtomicDataReader(std::string filepath) : m_data(2), m_filepath(filepath) {}
+  AtomicDataReader() = delete;
+  std::vector<std::vector<double>> GetData() { return m_data; }
+  std::vector<double> GetRates() { return m_data[rate_idx]; }
+  std::vector<double> GetTemps() { return m_data[T_idx]; }
```

Comment:
Refactor to use snake case for function names, consistent with other NESO headers.
```suggestion
  std::vector<std::vector<double>> get_data() { return m_data; }
  std::vector<double> get_rates() { return m_data[rate_idx]; }
  std::vector<double> get_temps() { return m_data[T_idx]; }
```